### ISSUE TITLE
Why we have 0 value for lambda?

### ISSUE THREAD

**Issue by bao-dai** [dhruvramani/C2AE-Multilabel-Classification#5]
Hi Dhruvramani,

I've checked your code and found that in C2AE-Multilabel-Classification/model/src/network.py, line 79, we should:
`return tf.norm(Fx-Fe)`
instread of:
`return tf.reduce_mean(tf.square(Fx - Fe))`

Since in the paper, the authors defined the loss as the Frobenius norm of Fx-Fe (the equation 2).
tf.reduce_mean just compute the expected value from all the elements of the input.
What do you think in this case 🍡 ?

**Comment by denyoblur** [comment]
Also in C2AE-Multilabel-Classification/model/src/network.py, line 106:
`lamda = 0.00`
why we only have 0 value for lambda?
